frankenphp.go

```diff
@@ -477,30 +457,24 @@ func ServeHTTP(responseWriter http.ResponseWriter, request *http.Request) error
 	fc.responseWriter = responseWriter
 	fc.startedAt = time.Now()
 
-	isWorker := fc.responseWriter == nil
-
 	// Detect if a worker is available to handle this request
-	if !isWorker {
-		if worker, ok := workers[fc.scriptFilename]; ok {
-			metrics.StartWorkerRequest(fc.scriptFilename)
-			worker.handleRequest(request)
-			<-fc.done
-			metrics.StopWorkerRequest(fc.scriptFilename, time.Since(fc.startedAt))
-			return nil
-		} else {
-			metrics.StartRequest()
-		}
+	if worker, ok := workers[fc.scriptFilename]; ok {
+		metrics.StartWorkerRequest(fc.scriptFilename)
+		worker.handleRequest(request)
+		<-fc.done
+		metrics.StopWorkerRequest(fc.scriptFilename, time.Since(fc.startedAt))
+		return nil
```

Comment:
nit: it's probably more readable to just do something like:

```go
if worker, ok := workers[fc.scriptFilename]; ok {
  return worker.handleRequest(fc, request)
}
```

That's a tiny little thing that has bothered me for awhile, just never had an excuse to change it. :)